magento/inventory
# Introduce API interfaces for Assigning Stocks to Sales channels #151

### Description
Created additional table to store stock channel items for msi inventory mapping.

### Related Issues 
#151 

### Manual testing scenarios
1. Install InventorySales and InventorySalesApi Module and check if 'inventory_stock_sales_channel' table was created.

### Contribution checklist
